### ISSUE TITLE
feddg (ELCFS) meta learning implementation is wrong

### ISSUE THREAD

**Issue by mynameisissac** [JeremyCJM/CCST#10]
https://github.com/JeremyCJM/CCST/blob/87c3e448cfa28c366b58d17f8dffc5d3d83574e5/federated/fed_run.py#L112-L122

Your meta learning implementation is wrong, `load_state_dict()` does not preserve the computation graph so `loss_outer` will never propagate gradient back to the original model parameters. be careful :)
